ingest/src/test/java/com/microsoft/azure/kusto/ingest/StreamingIngestClientTest.java

```diff
@@ -456,7 +457,7 @@ void IngestFromFile_JsonWrongMappingKind_IngestionClientException() {
         IngestionClientException ingestionClientException = assertThrows(IngestionClientException.class,
                 () -> streamingIngestClient.ingestFromFile(fileSourceInfo, ingestionProperties),
                 "Expected IngestionClientException to be thrown, but it didn't");
-        assertTrue(ingestionClientException.getMessage().contains("Wrong ingestion mapping for format 'json'; found 'Csv' mapping kind."));
```

Comment:
Add some more negative tests for the new rules

Reply:
I just checked, and see that the 2 new rules are ones that we just discovered we don't need. So I removed them, and now there are 0 new validation rules.

Reply:
Did you push it?